src/content/docs/accounts/original-accounts-billing/product-based-pricing/trial-lite-accounts-deprecated.mdx

```diff
@@ -64,7 +64,7 @@ Trial lengths depend on the product:
 
     <tr>
       <td>
-        [Browser](https://newrelic.com/products/browser-monitoring/pricing)
+        Browser
```

Comment:
This is a doc about deprecated pricing, so I assume it's ok to not change wording to "Browser monitoring," "Infrastructure monitoring," "Mobile monitoring," etc?

Reply:
Yeah, I just left those because it's about old pricing model and so I assume those things have special meaning.